RecoEgamma/EgammaElectronAlgos/src/TrajSeedMatcher.cc

```diff
@@ -54,7 +54,13 @@ edm::ParameterSetDescription TrajSeedMatcher::makePSetDescription()
   cutsDesc.add<double>("dRZMaxLowEtThres",20.);
   cutsDesc.add<std::vector<double> >("dRZMaxLowEtEtaBins",std::vector<double>{1.,1.5});
   cutsDesc.add<std::vector<double> >("dRZMaxLowEt",std::vector<double>{0.09,0.15,0.09});
-  desc.addVPSet("matchingCuts",cutsDesc);
+  edm::ParameterSet defaults;
+  defaults.addParameter<double>("dPhiMax",0.04);
+  defaults.addParameter<double>("dRZMax",0.09);
+  defaults.addParameter<double>("dRZMaxLowEtThres",0.09);
+  defaults.addParameter<std::vector<double> >("dRZMaxLowEtEtaBins",std::vector<double>{1.,1.5});
+  defaults.addParameter<std::vector<double> >("dRZMaxLowEt",std::vector<double>{0.09,0.09,0.09});
+  desc.addVPSet("matchingCuts",cutsDesc,std::vector<edm::ParameterSet>{defaults,defaults,defaults});
```

Comment:
There are three hits. Realizing now I should have made the first one different but it doesnt matter too much, one can always reset the cuts in confdb. So the default values are not perfectly sensible, if its a problem I can change it.